### PR TITLE
updating variable exercise 2

### PR DESCRIPTION
**Pull request by aaustin56**
updating variable exercise 2.

### CODE REVIEW

File: exercises/variable_exercise_2.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+weight = 150.1
+print("my weight is" + " " + str(weight))
+
+weight = weight - 35
+
+print("my goal weight is" + " " + str(weight))
+
+print(7 - 4 / 2)
+print(type(7 - 4 / 2))
```
Comment (raymendozajr) [inline exercises/variable_exercise_2.py]: Github suggests adding a new line at the end of the file

File: exercises/variable_exercise_2.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
+#integer
+x = 5
+print(x)
+
+
```
Comment (raymendozajr) [inline exercises/variable_exercise_2.py]: Just an FYI, so extra empty lines are allowed in python and do sometimes add to clarity, but in general we do not want to have more than one extra empty line.  We will talk about formatting after this.  If you have time I would suggest checking out a python auto-formatter called `Black`.

Additionally, I would also install Pycharm, which is a professional IDE (use the community version, it's free).  I can help orient you on that later as well.

File: exercises/variable_exercise_2.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+# ** (exponent)
+# 2 ** 3 gives us 8 
+
+
+print("my new height is" + str(height))
+
+print(3 + 4 / 2)
+print(type(3 + 4 / 2))
+
+
```
Comment (raymendozajr) [inline exercises/variable_exercise_2.py]: ditto with extra whitespace.

**Review by raymendozajr — CHANGES_REQUESTED** [review]
Just a few changes.  We will have to set up a CICD (continuous integration / continuous deployment) to verify that your code runs.  For now I have checked it manually and it ran with flying colors and no syntax errors!  Great job!

File: exercises/variable_exercise_2.py (hunk 2 above)
Comment (aaustin56) [reply exercises/variable_exercise_2.py]: OK good to know.

**Review by aaustin56 — COMMENTED** [review]
Awesome thanks for the feedback.

**Review by raymendozajr — APPROVED** [review]

**Pull request merged**